docs/overrides/main.html

```diff
@@ -1,5 +1,7 @@
 {% extends "base.html" %}
 
 {% block announce %}
+<a href="https://support.atlassian.com/contact/" title="Raise a support ticket" style="color:white"><strong>Contact Atlassian support &nbsp;&nbsp;&nbsp;</strong></a>
```

Comment:
I am not sure if we need this in the banner. I think customers should first read the support boundaries before contacting support. We can either send them to the support boundaries page or remove the link completely.

Potentially we can create a new page "How to ask for help" where we can list the options:
* community for general questions
* Atlassian support after making sure you fall within the boundaries
* what information to gather (link to troubleshooting)

Reply:
Agree